src/main/java/edu/ucla/library/bucketeer/Config.java

```diff
@@ -49,9 +49,12 @@
     public static final String S3_UPLOADER_THREADS = "s3.uploader.threads";
 
     // Where image files can be found on the local file system
-    public static final String FILESYSTEM_MOUNT = "bucketeer.fs.mount";
+    public static final String FILESYSTEM_IMAGE_MOUNT = "bucketeer.fs.image.mount";
```

Comment:
I'm undoubtedly missing something, but: why replace the constants instead of adding the new ones in parallel?

Reply:
This is just a renaming -- because now there are two "filesystem mounts", I thought it would be good to modify the name of this existing one (i.e., add "image" to it) to distinguish it from the "CSV" mount.